modules/generateMarkup.js

```diff
@@ -0,0 +1,94 @@
+import handleLike from './handleLike.js';
+import itemsCount from './itemCount.js';
+import displayPopup from './displayPopup.js';
+import displayLikes from './displayLikes.js';
+
+const appId = 'daS11VuHj0e3k7bb2TZc';
```

Comment:
The appId is a very confidential information hence should not be stored on github. Kindly consider putting it in a .env file or adding it to the additional information section when submitting your project for reviewing instead of exposing it here. :wink: